### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by wellemut**

### CODE REVIEW

File: locales/uk/home.json [hunk 1]
```diff
@@ -0,0 +1,145 @@
+{
+  "landingPage": {
+    "menu": {
```
Comment (navid-kalaei) [inline locales/uk/home.json:3]: why there are Russian strings in the UK locale?

File: locales/uk/map.json [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+  "embed": "Embed",
+  "burgerMenu": {
+    "home": "Home",
+    "join": "Join",
+    "loginForPilots": "Login",
+    "contact": "Contact",
+    "aboutUs": "About",
+    "privacy": "Privacy"
+  },
+  "mapColorMode": {
```
Comment (navid-kalaei) [inline locales/uk/map.json:357]: Russian strings in the UK locale?

**Review by navid-kalaei — COMMENTED** [review]
the are mistranslations in the uk strings

File: locales/uk/home.json (hunk 1 above)
Comment (wellemut) [reply locales/uk/home.json:3]: This is Ukraine file
Not en

**Comment by wellemut** [comment]
> the are mistranslations in the uk strings

No, UK stands for Ukraine

**Review by navid-kalaei — APPROVED** [review]

**Pull request merged**